scripts/download-and-extract-fixtures.sh

```diff
@@ -4,29 +4,38 @@
 #
 # Downloads execution spec test fixtures for zkevm.
 # By default, it fetches the latest release tag starting with 'zkevm@'.
-# An optional argument can be provided to specify an exact tag.
```

Comment:
When I did the tests, always pulling EEST fixtures using the script (that I reverted for the reasons mentioned before), I extended the script to also accept a target directory instead of a fixed one.

This feature isn't used in today's tests, but it remains valuable for the script.